### ISSUE TITLE
Show summary of current cycle within the explorer

### ISSUE THREAD

**Issue by devinbileck** [bisq-network/bisq-explorer#4]
Might be good to show a summary of the current cycle within the explorer, essentially all info from the governance dashboard. That way people can quickly check on the status without having to launch their client. Just a thought :)
![image](https://user-images.githubusercontent.com/603793/52694833-d1b06200-2f1f-11e9-8ea2-933320592a23.png)

**Comment by sqrrm** [comment]
I agree. Should be an easy task, just adding some extra data to the dump and display it under the stats tab. Anyone is welcome to take it on.
